app/views/security_group/new.html.haml

```diff
@@ -1,40 +1,44 @@
-%form#form_div{:name => "angularForm", 'ng-controller' => "securityGroupFormController"}
+%form#form_div{:name           => "angularForm",
+   "ng-controller" => "securityGroupFormController as vm",
+   "miq-form"      => true,
+   "model"         => "vm.securityGroupModel",
+   "model-copy"    => "vm.modelCopy"}
+
   = render :partial => "layouts/flash_msg"
+
   %h3
     = _('Network Management Provider')
   .form-horizontal
-    .form-group
+    .form-group{"ng-class" => "{'has-error': angularForm.name.$invalid}"}
       %label.col-md-2.control-label
         = _('Network Manager')
       .col-md-8
         = select_tag("ems_id",
-                      options_for_select([["<#{_('Choose')}>", nil]] + @ems_choices.sort),
-                      "ng-model"                    => "securityGroupModel.ems_id",
-                      "required"                    => "",
-                      :miqrequired                  => true,
-                      :checkchange                  => true,
-                      "ng-change"                   => "filterNetworkManagerChanged(securityGroupModel.ems_id)",
-                      "selectpicker-for-select-tag" => "")
+                     options_for_select([["<#{_('Choose')}>", nil]] + @ems_choices.sort),
+                     "ng-model"   => "vm.securityGroupModel.ems_id",
+                     "ng-change"  => "vm.filterNetworkManagerChanged(vm.securityGroupModel.ems_id)",
+                     :required    => "",
+                     "pf-select" => true)
+
   = render :partial => "common_new_edit"
+
   %h3
     = _('Placement')
   .form-horizontal
```

Comment:
Include the `Placement` label under the `ng-if` below since it's a label meant for `Cloud Tenant`